llvm/test/Transforms/InstCombine/zext-or-icmp.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN: opt < %s -passes=instcombine -S | FileCheck %s
+; RUN: opt < %s -passes='instcombine<no-verify-fixpoint>' -S | FileCheck %s
```

Comment:
We no longer reach fixed point because we end up unlocking a transform by creating a pattern we handle in knownbits.

If this is important, the fix here is to support sext/zext in the AssumptionCache.